### DISCUSSION TITLE
Docusaurus seems to be loading node_modules as part of docs when deployed on Vercel

### DISCUSSION THREAD

**Discussion by evisdrenova** [facebook/docusaurus#9366] [General]
Hey all, I'm running into an issue when I deploy docusaurus on vercel.  

The site builds fine and I can run it locally, but when I go to deploy it on Vercel, it doesn't work. It throws these warnings at the top of the vercel logs. 

```
[WARNING] Docs markdown link couldn't be resolved: (docs/definition-syntax.md) in "/vercel/path1/docs/node_modules/css-tree/README.md" for version current
10:37:52.882 | [WARNING] Docs markdown link couldn't be resolved: (docs/definition-syntax.md) in "/vercel/path1/docs/node_modules/css-tree/README.md" for version current
10:37:52.882 | [WARNING] Docs markdown link couldn't be resolved: (docs/definition-syntax.md) in "/vercel/path1/docs/node_modules/css-tree/README.md" for version current
10:39:14.372
```

Then later on in the log lines throws a lot of these errors (1600 lines):

```
SyntaxError: /vercel/path1/docs/node_modules/@alloc/quick-lru/readme.md: Expected corresponding JSX closing tag for <br>. (89:1)
--
10:39:14.456 | 87 \|     <sub>
10:39:14.456 | 88 \|         Tidelift helps make open source sustainable for maintainers while giving companies<br>assurances about security, maintenance, and licensing for their dependencies.
10:39:14.456 | > 89 \|     </sub>
10:39:14.456 | \|     ^
10:39:14.456 | 90 \| </div>
10:39:14.456 | 91 \|     </MDXLayout>
10:39:14.456 | 92 \|   )
10:39:14.456 | SyntaxError: /vercel/path1/docs/node_modules/@babel/code-frame/node_modules/ansi-styles/readme.md: Unterminated JSX contents. (122:16)
10:39:14.456 | 120 \| <h2 {...{"id":"license"}}>{License}</h2>
10:39:14.456 | 121 \| <p>{MIT}</p>
10:39:14.456 | > 122 \|     </MDXLayout>
10:39:14.456 | \|                 ^
10:39:14.456 | 123 \|   )
10:39:14.456 | 124 \| };
10:39:14.456 | 125 \| MDXContent.isMDXComponent = true
```

And this is repeated for every readme.md file we have in our node_modules. It looks like docusaurus is trying to load node_modules but I'm ignoring it in my .gitignore.

```
# Dependencies
/node_modules/

# Production
/build

# Generated files
.docusaurus
.cache-loader

# Misc
.DS_Store
.env.local
.env.development.local
.env.test.local
.env.production.local

npm-debug.log*
yarn-debug.log*
yarn-error.log*
```

How can I resolve this?

Here is a link to the branch: https://github.com/nucleuscloud/neosync/tree/more-usecases

**Comment by evisdrenova** [comment]
Doing further research and this seems to be a vercel issue when using a monorepo (like we are). My current workaround is to have vercel build the project every time and **not** ignore the build step. Here is a ticket on vercel that seems to be open (https://github.com/orgs/vercel/discussions/285). When I redeploy a branch that failed in Vercel with the errors above and have vercel build the project, it works fine.

**Comment by slorber** [comment]
It's not possible to help because your repro branch link doesn't work unfortunately.

I don't see anything wrong on main here: https://github.com/nucleuscloud/neosync and I don't know how your branch is different. If only your branch fails then it's important to study the diff, so show me a PR.

I don't understand why you think the Vercel ticket is related 😅 but great for you if you found a workaround
